dbt/adapters/oracle/impl.py

```diff
@@ -98,18 +99,13 @@ def convert_number_type(cls, agate_table, col_idx):
     def convert_time_type(cls, agate_table, col_idx):
         return "timestamp"
 
-    @available
-    def verify_database(self, database):
```

Comment:
This check makes no sense as no matter if one connects using `OracleConnectionMethod` -> `HOST`, `TNS` or `CONNECTION_STRING`, you will always be on the same database (`select SYS_CONTEXT('userenv', 'DB_NAME') from dual;`) and therefore the exception will never be thrown.